tests/fsharp/typecheck/sigs/neg60.fs

```diff
@@ -85,3 +85,115 @@ module NullableOperators3 =
     let a = new System.Nullable<int>(0)
     let b = new System.Nullable<int>(1)
     if a ?=? b then printfn "Same" else printfn "Differerent"
+
+module QuerySyntaxWithValidOverloading = 
+
+    open System
+
+    type Content = ArraySegment<byte> list
+
+    type ContentBuilder() =
+        member this.Run(c: Content) =
+            let crlf = "\r\n"B
+            [|for part in List.rev c do
+                yield! part.Array.[part.Offset..(part.Count+part.Offset-1)]
+                yield! crlf |]
+
+        member this.Yield(_) = []
+
+        [<CustomOperation("body")>]
+        member this.Body(c: Content, segment: ArraySegment<byte>)  =
+            segment::c
+
+        [<CustomOperation("body")>]
+        member this.Body(c: Content, bytes: byte[], offset, count) =
+            ArraySegment<byte>(bytes, offset, count)::c
+
+        [<CustomOperation("body")>]
+        member this.Body(c: Content, [<ParamArray>] contents: string[]) =
+            let rec loop acc (contents: string list) =
+                match contents with
+                | [] -> acc
+                | content::rest ->
+                    let bytes = Text.Encoding.ASCII.GetBytes(content)
+                    loop (this.Body(c, bytes, 0, bytes.Length)) rest
+            loop c (List.ofArray contents)
```

Comment:
Do these count as different arity or am I missing something?

Reply:
My bad, seems these were added in https://github.com/dotnet/fsharp/pull/9400/commits/c6f4e47ca671bccee5f49e006f7447cc339e9f48 which I had overlooked!